### ISSUE TITLE
If you want some help I have some tools that can probably help you !

### ISSUE THREAD

**Issue by lilili87222** [Dadoum/Provision#7]
If you want some help I have some tools that can probably help you !

Do you mean this project?

**Comment by Dadoum** [comment]
Yes. Actually you can hook any function and see how it affects CPIM generation, what's fed to vdfut thingy, what it calls, etc...

**Comment by lilili87222** [comment]
really? can we contact with email ,or other im tool ?

**Comment by Dadoum** [comment]
Do you have a Discord account ?

**Comment by lilili87222** [comment]
Yes ,how to join

**Comment by lilili87222** [comment]
I have send a request of discord friend
